src/apps/vpn/inspector/inspectorhandler.cpp

```diff
@@ -567,6 +567,24 @@ static QList<InspectorCommand> s_commands{
                        return obj;
                      }},
 
+    InspectorCommand{"messages", "Returns a list of the loaded messages ids", 0,
```

Comment:
We already have a similar method for guides. Wondering if we should return all the add-ons with types.